mvnxparser/mvnxStreamReader/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright: (C) 2017 iCub Facility
+# Author: Diego Ferigo
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+# Load dependencies
+find_package(Qt5Xml REQUIRED)
+
+# Build the libraries
```

Comment:
I hardcoded it in the first tests I did, just removed it in all the files of this folder tree